### ISSUE TITLE
Multi-variate test in Plink

### ISSUE THREAD

**Issue by SiyangLiu** [chrchang/plink-ng#110]
Hi Chang, 
Does Plink support multiple phenotype association test?

Best,
Siyang

**Comment by chrchang** [comment]
No, and this isn't likely to change soon; use other software for this.  Maybe several years down the line, though.
